config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_vertexaidatasets.vertexai.cnrm.cloud.google.com.yaml

```diff
@@ -67,11 +67,215 @@ spec:
                   Dataset. If set, this Dataset and all sub-resources of this Dataset
```

Comment:
I'm curious - can `encryptionSpec` be used by itself (as an empty struct)? The reason I asked is because the field description looks a bit confusing. I want to avoid we run into a similar issue like b/325359109.

Reply:
https://cloud.google.com/vertex-ai/docs/reference/rest/v1/EncryptionSpec
Says this field "Represents a customer-managed encryption key spec that can be applied to a top-level resource." I don't think an empty struct is meaningful for this field.